<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="Start_date">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="stop_date">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Start_frequency">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>104</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="stop_frequency">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>104</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="Full_Range">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Range</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SpecifiedRange">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Specified Range</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Prime Focus 1 340MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prime Focus 1 450 MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prime Focus 1 600MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prime Focus 1 800MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prime Focus 2 1070MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rcvr1_2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rcvr2_3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rcvr4_6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rcvr8_10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ku Band</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K Band Focal Plane Array</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ka Band</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q Band</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>W Band</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Receiver List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>680</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Save_Data">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Directory path here:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="rfi_file">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>630</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>590</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter file name here:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
